trunk/cli/src/commands/pgx.rs

```diff
@@ -1,36 +1,21 @@
-use bollard::container::{
```

Comment:
In the PGX logic, we delete everything that is not reusable and move reusable parts into containers.rs. Then we use the reusable logic that is shared with C builder in this builder.